packages/hardhat-cannon/src/builder/index.ts

```diff
@@ -311,31 +362,69 @@ export class ChainBuilder {
 
     const sortedFileList = _.sortBy(
       fileList
-        .filter((n) => /^[0-9]*-.*.json$/.test(n))
+        .filter((n) => /^[0-9]*-[0-9]*.json$/.test(n))
```

Comment:
Not sure if this could cause a problem, but, this regex is allowing files named `-.json` and the dot on `.json` is not scaped, here's a simple fix:
```suggestion
        .filter((n) => /^[0-9]+-[0-9]+\.json$/.test(n))
```

Reply:
ab33645